In [42]:
import json 
import pandas as pd

In [43]:
data=pd.read_json(r"C:\Users\venuk\OneDrive\Desktop\facebook\likes_and_reactions\pages.json")

In [45]:
data  

page_likes
0    {'name': 'Applied AI Course', 'timestamp': 159...
1       {'name': 'Homebliss', 'timestamp': 1581146999}
2    {'name': 'Weekend Thrill', 'timestamp': 156497...
3    {'name': 'BEARD MAN KING', 'timestamp': 156367...
4     {'name': 'Telugu Swag', 'timestamp': 1551952328}
..                                                 ...
370         {'name': 'Kaká', 'timestamp': 1385778054}
371  {'name': 'Pawan Kalyan', 'timestamp': 1385777916}
372  {'name': 'A. P. J. Abdul Kalam', 'timestamp': ...
373  {'name': 'Swami Vivekananda', 'timestamp': 137...
374        {'name': 'Shewag', 'timestamp': 1376923498}

[375 rows x 1 columns]

In [46]:
data1=pd.read_json(r"C:\Users\venuk\OneDrive\Desktop\facebook\likes_and_reactions\posts_and_comments.json")

In [50]:
data1.keys()

Index(['reactions'], dtype='object')

In [68]:
data1['reactions'][0]

{'timestamp': 1599705750,
 'data': [{'reaction': {'reaction': 'HAHA', 'actor': 'Venukiran Sankar'}}],
 'title': "Venukiran Sankar reacted to Durga Prasad's post."}

In [69]:
len(data1)

3281

total no reactions  for  post and comments is 3281

total nof reactions for pages is 375